app/src/main/java/com/sw_ss16/lc_app/ui/base/BaseActivity.java

```diff
@@ -67,6 +68,8 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        //Check if first startup, if yes
```

Comment:
Should this be a TODO comment? Since we don't actually check for first startup yet.